components/org.wso2.carbon.identity.application.authenticator.oidc/pom.xml

```diff
@@ -175,7 +184,9 @@
                             version="${carbon.identity.inbound.oauth.package.import.version.range}",
                             org.wso2.carbon.idp.mgt; version="${identity.framework.package.import.version.range}",
                             org.wso2.carbon.identity.application.common.util;
-                            version="${identity.framework.package.import.version.range}"
+                            version="${identity.framework.package.import.version.range}",
+                            org.wso2.carbon.identity.central.log.mgt.utils;
+                            version="${identity.framework.package.import.version.range}",
```

Comment:
let's bump the import version range as well

Reply:
Fixed with 9454f89b9ba4bfe27948b6fa1bbfe6a875db3bb1